tests/models/test_dag.py

```diff
@@ -2623,6 +2624,29 @@ def test_replace_outdated_access_control_actions(self):
             dag.access_control = outdated_permissions
         assert dag.access_control == updated_permissions
 
+    def test_validate_executor_field_executor_not_configured(self):
+        dag = DAG(
+            "test-dag",
+            schedule=None,
+        )
+
+        EmptyOperator(task_id="t1", dag=dag, executor="test.custom.executor")
+        with pytest.raises(
+            ValueError, match="The specified executor test.custom.executor for task t1 is not configured"
+        ):
+            dag.validate()
+
+    def test_validate_executor_field(self):
+        with patch.object(ExecutorLoader, "lookup_executor_name_by_str") as mock_get_executor_names:
+            dag = DAG(
+                "test-dag",
+                schedule=None,
+            )
+
+            mock_get_executor_names.return_value = None
```

Comment:
Do you need to bother specifying None as a return value here? The mock should just return a mock object by default when it's called which should do just as well, no?

Reply:
Yep, you're correct. Made the change. Thanks!